### ISSUE TITLE
Collectd data from remote hosts not rendering in graph

### ISSUE THREAD

**Issue by pamolloy** [facette/facette#308]
I am running two instances of collectd, each on a different host. I have configured the network plugin on each host to send from the remote instance to a local one running on my laptop. The remote data seems to be written locally, but Facette doesn't render that data (e.g. cpu and load). Note that I am able to render local data (e.g. load). The remote shows up on the Origins page and I am able to create a graph with the aforementioned metrics. I have also confirmed the following:

- Confirmed I am receiving packets from the remote instance of collectd using `tcpdump`
- Followed collectd's logging output to Systemd's journal (e.g. ```write_log values: remote.cpu-1.cpu-system 60511 1469542643```)
- `rrdtool dump remote/cpu-1/cpu-system.rrd` contains some data
- Permissions of the `*.rrd` files are the same

When I run `facette -c docs/examples/facette.json -L debug` I see the following for _both_ the remote and local hosts.

```
2016/11/11 08:50:38 DEBUG: catalog: appending metric `load.shortterm' to source `remote' via origin `collectd'
2016/11/11 08:50:38 DEBUG: server: discard record {Origin: "collectd", Source: "remote", Metric: "load/load/shortterm/min"}, as metric doesn't match `/average$' sieve pattern
2016/11/11 08:50:38 DEBUG: server: discard record {Origin: "collectd", Source: "remote", Metric: "load/load/shortterm/max"}, as metric doesn't match `/average$' sieve pattern
2016/11/11 08:50:38 DEBUG: server: discard record {Origin: "collectd", Source: "remote", Metric: "load/load/midterm/min"}, as metric doesn't match `/average$' sieve pattern
2016/11/11 08:50:38 DEBUG: server: discard record {Origin: "collectd", Source: "remote", Metric: "load/load/midterm/max"}, as metric doesn't match `/average$' sieve pattern
2016/11/11 08:50:38 DEBUG: catalog: appending metric `load.midterm' to source `remote' via origin `collectd'
2016/11/11 08:50:38 DEBUG: catalog: appending metric `load.longterm' to source `remote' via origin `collectd'
2016/11/11 08:50:38 DEBUG: server: discard record {Origin: "collectd", Source: "remote", Metric: "load/load/longterm/min"}, as metric doesn't match `/average$' sieve pattern
2016/11/11 08:50:38 DEBUG: server: discard record {Origin: "collectd", Source: "remote", Metric: "load/load/longterm/max"}, as metric doesn't match `/average$' sieve pattern
```

I have specified a directory in my home directory as the `data_dir` as well as to store `facette.pid`. I also removed all providers except for collectd from the providers directory.

I would not be surprised if I missed something in the GUI. Here is the corresponding JSON generated from graph I created:
```{
    "id": "5328ee0a-e7e5-4d0b-5673-76795a4bcdf4",
    "name": "Remote CPU",
    "description": "", 
    "type": 1,
    "stack_mode": 1,
    "unit_type": 1,
    "groups": [
        {
            "name": "0.system",
            "type": 1,
            "stack_id": 0,
            "series": [
                {
                    "name": "0.system",
                    "origin": "collectd",
                    "source": "remote",
                    "metric": "cpu.0.system",
                    "options": null
                }
            ],
            "options": {}
        },
        {
            "name": "1.system",
            "type": 1,
            "stack_id": 0,
            "series": [
                {
                    "name": "1.system",
                    "origin": "collectd",
                    "source": "remote",
                    "metric": "cpu.1.system",
                    "options": null
                }
            ],
            "options": {}
        }
    ],  
    "template": false
}
```

**Comment by vbatoufflet** [comment]
Hi @pamolloy,

Is there any specific message recorded in the logs when you try to display the created graph?

If you run `rrdtool dump …/cpu-0/cpu-idle.rrd` command, are there values in the RRD file?

**Comment by pamolloy** [comment]
When running `facette` with `-L debug` I just see a bunch of HTTP requests being logged.

Yup, I had checked that earlier. I did one extra experiment that was interesting. I swapped `remote/load/load.rrd` and `local/load/load.rrd` and the graphs swapped (i.e. the local load graph displayed nothing, while the remote load graph displayed the local data correctly). Removing both displays `No data` as expected.

I dumped the remote RRD and uploaded it here: https://gist.github.com/pamolloy/511eb9b231fa21433a896c0ee5b7eab2
